src/plugins/numberline/components/numberline-tile.tsx

```diff
@@ -0,0 +1,347 @@
+import classNames from 'classnames';
+import { axisBottom, drag, pointer, scaleLinear, select } from 'd3';
+import { observer } from 'mobx-react';
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react';
+
+import { NumberlineToolbar } from "./numberline-toolbar";
+import { NumberlineContentModelType, PointObjectModelType,  } from "../models/numberline-content";
+import {
+  kAxisStyle, kAxisWidth, kContainerWidth, kNumberLineContainerHeight, numberlineDomainMax, numberlineDomainMin,
+  tickHeightDefault, tickHeightZero, tickStyleDefault, tickStyleZero, tickWidthDefault, tickWidthZero,
+  innerPointRadius, outerPointRadius, numberlineYBound, yMidPoint, kTitleHeight, kBoundingBoxOffset, kArrowheadTop,
+  kArrowheadOffset, kPointButtonRadius
+} from '../numberline-tile-constants';
+import { BasicEditableTileTitle } from "../../../components/tiles/basic-editable-tile-title";
+import { useToolbarTileApi } from "../../../components/tiles/hooks/use-toolbar-tile-api";
+import { ITileProps } from "../../../components/tiles/tile-component";
+import { OffsetModel } from '../../../models/annotations/clue-object';
+import { ITileExportOptions } from "../../../models/tiles/tile-content-info";
+import { HotKeys } from "../../../utilities/hot-keys";
+
+import "./numberline-tile.scss";
+
+export const NumberlineTile: React.FC<ITileProps> = observer(function NumberlineTile(props){
+  const { documentContent, model, readOnly, scale, tileElt, onRegisterTileApi, onUnregisterTileApi } = props;
+  const content = model.content as NumberlineContentModelType;
+  const [hoverPointId, setHoverPointId] = useState("");
+  const [_selectedPointId, setSelectedPointId] = useState(""); // Just used to rerender when a point is selected
+
+  // Basic model manipulation functions
+  const deleteSelectedPoints = useCallback(() => {
+    content.deleteSelectedPoints();
+  }, [content]);
+
+  const createPoint = (xValue: number) => {
+    if (!readOnly) {
+      const point = content.createNewPoint(xValue);
+      content.setSelectedPoint(point);
```

Comment:
Just a reminder to send the deployed link to Leslie and make sure she's okay with the following behavior: when you click on a point  it's automatically selected (blue) and has an outer blue hover circle.